src/autoscaler/configutil/cf.go

```diff
@@ -21,10 +24,16 @@ type VCAPConfigurationReader interface {
 	MaterializeDBFromService(dbName string) (string, error)
 	MaterializeTLSConfigFromService(serviceTag string) (models.TLSCerts, error)
 	GetServiceCredentialContent(serviceTag string, credentialKey string) ([]byte, error)
+
+	GetInstanceTLSCerts() models.TLSCerts
+
 	GetPort() int
+	GetSpaceGuid() string
+	GetOrgGuid() string
+	GetInstanceIndex() int
 	IsRunningOnCF() bool
 
-	ConfigureStoredProcedureDb(dbName string, confDb *map[string]db.DatabaseConfig, storedProcedureConfig *models.StoredProcedureConfig) error
+	ConfigureDb(dbName string, confDb *map[string]db.DatabaseConfig) error
```

Comment:
Check if method can be private and unify with configure databases

Reply:
fixed in a20861895